### PR TITLE
commented, printed and tested server.py

### PR DESCRIPTION
**Pull request by REGATTE**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -178,5 +178,6 @@ def isPrime(num):
 
 # Run function for connection
 recieve()
+print("receiving")
```
Comment (AbhinavRobinson) [inline server.py:181]: this outputs after code runs >.<

File: server.py [hunk 2]
```diff
@@ -3,36 +3,36 @@
...
 
  Project by Abhinav Robinson
+ Comments & Print statements by Ashok Kumar
 """
 
 
 # Import python socket, random, sympy, hashlib, pickle module
 import socket, random, sympy, hashlib, pickle
-
+print("initialised model")
```
Comment (AbhinavRobinson) [inline server.py:12]: printed before initialisation

File: server.py [hunk 3]
```diff
@@ -3,36 +3,36 @@
...
 
 # Connect and recieve message
 def recieve():
 
     # Define prime upper bound
     PRIME_LIMIT = 1000000000000
-
+    print("limit = 1000000000000")
```
Comment (AbhinavRobinson) [inline server.py:19]: add input() to take from user?

File: server.py [hunk 4]
```diff
@@ -3,36 +3,36 @@
...
     # Generate Key Pair
     e,d,n = generate(sympy.randprime(1,PRIME_LIMIT),sympy.randprime(1,PRIME_LIMIT))
-
+    print("keys generated")
```
Comment (AbhinavRobinson) [inline server.py:22]: okay

File: server.py [hunk 5]
```diff
@@ -3,36 +3,36 @@
...
     print(f"Encryption Key: {e} \n Mod Key: {n} \n Secret Key: {d}")
-
+ 
     # Initialize socket stream
     s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-
+    print("socket initialised")
```
Comment (AbhinavRobinson) [inline server.py:27]: okay

File: server.py [hunk 6]
```diff
@@ -3,36 +3,36 @@
...
     # Specify port
     port = 6000
-
+    print("post: 6000, binding to port")
```
Comment (AbhinavRobinson) [inline server.py:30]: rewrite -> "Listening to Port: 6000

**Review by AbhinavRobinson — CHANGES_REQUESTED** [review]
Make changes :)

**Comment by AbhinavRobinson** [comment]
Make changes and PR 🚨

File: server.py (hunk 6 above)
Comment (AbhinavRobinson) [reply server.py:30]: also use "port" variable declaired above

File: server.py (hunk 3 above)
Comment (REGATTE) [inline server.py:19]: PRIME_LIMIT = int(input("enter a value to define upper bound"))

**Pull request closed without merging**